framework/include/meshgenerators/SideSetsGeneratorBase.h

```diff
@@ -52,13 +52,46 @@ class SideSetsGeneratorBase : public MeshGenerator
    */
   void flood(const Elem * elem, Point normal, boundary_id_type side_id, MeshBase & mesh);
 
-  boundary_id_type getNextBoundaryID() const;
+  bool normalsWithinTol(const Point & normal_1, const Point & normal_2) const;
```

Comment:
```suggestion
  bool normals_within_tol(const Point & normal_1, const Point & normal_2) const;
```
camel case is for class names

Reply:
add docstring too

Reply:
oh you re right